### PR TITLE
UI

### PR DESCRIPTION
**Pull request by nikolaswise**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "js": "browserify -e source/js/index.js -d -o build/bundle.js",
-    "sass": "node-sass source/sass/ -q -o build/",
+    "sass": "node-sass source/sass/ --include-path node_modules/ -q -o build/ ",
```
Comment (paulcpederson) [inline package.json]: trailing whitespace?

Comment (nikolaswise) [inline package.json]: jerk.

Comment (paulcpederson) [inline package.json]: just bein a dick! :+1:

Comment (nikolaswise) [inline package.json]: Ill get there. Adding a button someone else

**Pull request merged**